sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/EventRoute.java

```diff
@@ -79,13 +77,13 @@ public EventRoute setFilter(String filter) {
     }
 
     /**
-     * Validates the instance.
+     * Sets the event route's Id.
      *
-     * @throws IllegalArgumentException thrown if the instance is not valid.
+     * @param id The event route's Id to set.
+     * @return the EventRoute object itself.
      */
-    public void validate() {
-        if (getEndpointName() == null) {
-            throw new IllegalArgumentException("Missing required property endpointName in model EventRoute");
-        }
+    public EventRoute setId(String id) {
```

Comment:
While users shouldn't be setting this, we need it to do conversions from generated type to public type

Reply:
The converters live in a different package, so I can't just make this package private

Reply:
How about a doc comment to that effect then? If I'm an end-user and see this setter, I'm wondering what happens if I set it, or why I'd want to.